src/main/java/io/codelex/flightplanner/controller/AdminController.java

```diff
@@ -0,0 +1,40 @@
+package io.codelex.flightplanner.controller;
+
+import io.codelex.flightplanner.flights.AddFlightRequest;
+import io.codelex.flightplanner.flights.Flight;
+import io.codelex.flightplanner.service.FlightInMemoryService;
+import io.codelex.flightplanner.service.FlightService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/admin-api")
+public class AdminController {
+
+    public FlightService service;
+
+    public AdminController(FlightService service) {
+        this.service = service;
+    }
+
+    @PutMapping("/flights")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Flight addFlight(@Valid @RequestBody AddFlightRequest flightRequest) {
+        Flight flight = flightRequest.requestToFlight(FlightInMemoryService.generateUUID());
```

Comment:
To konvertāciju no AddFlightRequest uz Flight labāk veic iekš service un droši padod pašam service AddFlightRequest. Tāpēc, ka tad, kad slēgsim klāt DB controller paliks tāds pats, bet ID ģenerēsies savādāk, tāpēc to tāpat nevarēs šeit atstāt.